EPCPortalTesting/DataHandlerModelTests.cs

```diff
@@ -0,0 +1,30 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using EPCPortalWeb;
+using EPCPortalWeb.Models;
+using System.Collections.Generic;
+
+namespace EPCPortalTesting
+{
+    [TestClass]
+    public class DataHandlerModelTests
+    {
+        DataHandlerModel dataHandlerModel = new DataHandlerModel();
+       
+        [TestMethod]
+        public void SearchForAddressByHouseNumber()
```

Comment:
It helps to phrase the test name using the "Method under test, When, Then" pattern - or something like it - so devs looking at it know what exactly is being tested and why it might be failing.

E.g., FilterAddressesByHouseNumber_WhenHouseNumberInListOfAddresses_FilteredListIsNotNull

It is more verbose, admittedly, but when the tests start to pile up, it makes it much easier to see exactly what is happening at a glance, via the TestRunner.